index.html

```diff
@@ -1492,6 +1492,65 @@ <h5 class="modal-title">Rules for Roots of Quadratic Equation</h5>
                             <input type="tel" class="form__field" id="coffc" placeholder="Enter Number"
                                 oninput="rootquadratic();">
                         </div>
+
```

Comment:
Here is some extra code which you were trying to commit, which was causing the merge conflict. Can you have a look at it again?

Reply:
But this is not my code

Reply:
Okay, then, you can remove the commented part and I'll approve the PR.

Reply:
I'm approving these changes as of now.